test/jest/mock-store.js

```diff
@@ -210,10 +210,12 @@ export const createSwapsMockStore = () => {
         },
       ],
       useCurrencyRateCheck: true,
-      currentCurrency: 'ETH',
+      currentCurrency: 'usd',
       currencyRates: {
         ETH: {
-          conversionRate: 1,
+          conversionDate: 1708532473.416,
+          conversionRate: 2918.02,
+          usdConversionRate: 2918.02,
```

Comment:
The purpose of the `createSwapsMockState` function is that it creates a mock state for Swaps that can be reused across different test files instead of duplicating mock data. E.g. `isGasIncludedTrade` is already used in ui/ducks/swaps/swaps.test.js and review-quote.test.js. Because of that I would prefer to keep this mock data here in one place for reusability and overwrite it in individual tests if needed.